scripts/variantstore/TERRA_QUICKSTART.md

```diff
@@ -21,38 +21,53 @@ This quickstart assumes that you are familiar with Terra workspaces, the data mo
     - BigQuery Read Session User
 4. These tools expect re-blocked gVCF files as input.
 
-## 1. Import Data
-In order to load data into BigQuery without hitting daily load limits, we recommend you group your input files into samples sets and follow these steps for each sample set.
+## 1. Setup workspace with data model
+This workspace has two files you can use to initially populate the data model. Both files are in the `Files` section of the workspace. One named `samples_10.tsv` has 10 samples with already reblocked gvcfs. The other file `samples_3202.tsv` has 3202 samples only a few of which have their reblocked gvcf attributes filled in. 
 
-A sample set for the quickstart has already been created with 10 samples and paths to re-blocked gVCFs for each sample.  Run the two import workflows against this sample set by selecting "sample_set" as the root entity type ("Step 1") and `gvs_demo-10` for the data ("Step 2").  If you are creating your own sample set, note that the sample table should have a column for the re-blocked gVCFs (`hg38_reblocked_gvcf` or `reblocked_gvcf_path`) and their index files need to be in the same location.
+## 2. Reblock samples
+Run the GvsAoUReblockGvcf workflow on the samples
+
+These are the required parameters which must be supplied to the workflow:
+
+| Parameter      | Description |
+| ----------------- | ----------- |
+| gvcf | The gvcf file to reblock |
+| ref_dict      		| The reference dictionary to use       |
+| ref_fasta      	| The reference to use      |
+| ref\_fasta_index   | The reference index       |
 
-## 1.1 Assign Gvs IDs
-To optimize the internal queries, each sample must have a unique and consecutive integer ID assigned. Run the `GvsAssignIds` workflow, which will create an appropriate ID for each sample in the sample set and update the BigQuery dataset with the sample name to ID mapping info.
+
+## 3. Load data
+### 3.1 Assign Gvs IDs (suggest renaming to GvsPrepareSamples)
+To optimize the internal queries, each sample must have a unique and consecutive integer ID assigned. Run the `GvsAssignIds` workflow on a sample set, which will create an unique gvs id, update the BQ sample\_info table (creating it if it doesn't exists), and update the data model. This workflow takes care of creating the BQ vet and ref_ranges tables needed for the sample ids generated.
 
 These are the required parameters which must be supplied to the workflow:
 
 | Parameter      | Description |
 | ----------------- | ----------- |
 | project_id | The name of the google project containing the dataset |
 | dataset_name      | The name of the dataset you created above       |
+| external\_sample_names      | Sample ids from the data model that will be mapped to gvs ids       |
+| workspace_namespace      | The namespace of the workspace       |
```

Comment:
This part feels funky -- maybe there's not a better way to do it but this means you must run this workflow inside Terra (thinking of other users) but also IF you make a mistake and put the wrong workspace name here… it could mess up your data in the other workspace.

Is there not a way to map these results back onto the data model using Terra features?  If not.. maybe we should rethink about how IDs are allocated. 

ToL: why do we need these IDs in the data model anyway?  Feels like an implementation detail we shouldn't expose our users to…

Reply:
Again, I meant for this documentation to be specific to the workspace we use for AoU. I think it is useful to have this information in the workspace (just given how many times I've had to get the mapping between the two for tracking - but maybe we won't have the same issues that I had to deal with previously). It can be removed if it really isn't helpful.